src/yaku.js

```diff
@@ -63,7 +68,8 @@ function getYaku(inputHand, inputCalls) {
 	//Sanankou
 	//3 concealed triplets
 	//Open*
-	var sanankou = getSanankou(inputHand);
+	let sanankou = getSanankou(triplets);
```

Comment:
This is not going to work: `triplets` cointains also the triplets from calls/melds. That means by using this variable the function will also check for open triplets which is basically the same as Toitoi.

Reply:
Yes, but this PR is only about the "help" mode, so I will temporarily remove this part of the logic, I will test this part of the logic more thoroughly before making another PR. ;-)